src/reference/asciidoc/whats-new.adoc

```diff
@@ -33,6 +33,10 @@ See <<./reactive-streams.adoc/reactive-message-handler,ReactiveMessageHandler>>
 `spring-integration-mongodb` module now provides channel adapter implementations for Reactive MongoDB driver support in Spring Data.
 See <<./mongodb.adoc#mongodb-reactive-channel-adapters,MongoDB Reactive Channel Adapters>> for more information.
 
+[[x5.3-AbstractCorrelatingMessageHandler]]
+==== AbstractCorrelatingMessageHandler(Splitter/ScatterGather)
+Starting with version 5.3, if `MessageGroupProcessor` returns a Message, `MessageBuilder.popSequenceDetails()` will be performed on the `outputMessage`  if the sequenceDetails matches with first message of group.
```

Comment:
There has to be a blank like after a title.

The title is slightly wrong. Would be better to say: `Aggregator Changes`. Your one doesn't give an clue what I'm to read about.

Also consider to add a like to the chapter. See other sections here for samples.